okhttp/src/main/java/okhttp3/RealCall.java

```diff
@@ -65,6 +66,10 @@ protected RealCall(OkHttpClient client, Request originalRequest) {
     }
   }
 
+  private void captureCallStackTrace() {
```

Comment:
Apologies - I wanted to make sure the approach was ok before I went back and started renaming things. I'll clean this up.